subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/horreum_lts_store.py

```diff
@@ -112,16 +123,18 @@ def build_lts_payloads() -> dict:
                 "settings": _parse_entry(entry.settings)
             }
         }
-        
+
         yield output, start_time, end_time
 
 
-def _decode_users(users, hostnames, pod_times):
+def _decode_users(results):
```

Comment:
I first changed the name of the arguments
`users`, `hostnames`, `pod_times` into `ods_ci`, `testpod_hostnames`, `notebook_pod_time`

so that they are consistent with `results.<name>`. It makes the grepping and troubleshooting easier if objects always have the same (or similar, like `src_<name>`) naming